File: src/main/java/model/data/RepoRequest.java
package model.data;

import model.repo.Repo;

public class RepoRequest extends Data {

    private Repo repo;
    private String link;

    public RepoRequest(Repo repo, String link) {
        this.repo = repo;
        this.link = link;
    }

    public Repo getRepo() {
        return repo;
    }

    public String getLink() {
        return link;
    }

}

File: src/main/kotlin/com/mirf/features/repositoryaccessors/data/RepoRequest.kt
package com.mirf.features.repositoryaccessors.data

import com.mirf.core.data.MirfData
import com.mirf.core.repository.RepositoryCommander

/**
 * Class used by Repository accessors. Contains information about repository and link for request
 */
open class RepoRequest(val link: String,
                       val repositoryCommander: RepositoryCommander,
                       var bundle: Any? = null) : MirfData()
